package.json

```diff
@@ -1,15 +1,11 @@
 {
   "name": "@essex/pbi-base",
-  "version": "0.9.0",
+  "version": "1.0.0",
+  "private": true,
```

Comment:
We're using a new feature of yarn called "workspaces". The `private: true` just makes it abundantly clear that we are not publishing the top level package, but only the nested packages under the `packages` directory.  https://yarnpkg.com/blog/2017/08/02/introducing-workspaces/#setting-up-workspaces. We will no longer be publishing a `@essex/pbi-base` package.